framework/src/main/java/com/github/dbmdz/flusswerk/framework/flow/Flow.java

```diff
@@ -31,7 +31,6 @@ public class Flow {
   private final Function<Object, Collection<Message>> writer;
   private final Runnable cleanup;
   private final Set<Consumer<FlowInfo>> flowMetrics;
-  private final Tracing tracing;
 
   public Flow(FlowSpec flowSpec, Tracing tracing) {
```

Comment:
Do we still need the `tracing` parameter here?

Reply:
Thank you, it's gone now.